package.json

```diff
@@ -10,10 +10,13 @@
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
+    "@typescript-eslint/eslint-plugin": "^2.29.0",
```

Comment:
it already was present in pack of @shelf/eslint-config

so in current config  eslint extends this pack 
![image](https://user-images.githubusercontent.com/13865639/80274072-7449a080-86e0-11ea-9379-838a9d2ed68d.png)

that had such config inside

![image](https://user-images.githubusercontent.com/13865639/80274058-6005a380-86e0-11ea-96cc-44b4601c5e45.png)

but it's ok if we have in separately in json

Reply:
yes i tried to resolve an error with it :) but it was in node version incompatibility